src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -261,7 +261,7 @@ class WorkflowManagerActor(backend: Backend) extends LoggingFSM[WorkflowManagerS
   }
 
   private def assertCallExistence(id: WorkflowId, callFqn: FullyQualifiedName): Future[Any] = {
-    globalDataAccess.getExecutionStatus(id, ExecutionDatabaseKey(callFqn, None)) map {
+    globalDataAccess.getExecutionStatus(id, ExecutionDatabaseKey(callFqn, None, 1)) map {
```

Comment:
This method is used to respond to API queries which don't support the concept of attempt (and probably shouldn't). It verifies that a call with this FQN and index exists, which should be true if there was at least one attempt so I think 1 is ok here.